doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -48,18 +49,42 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
+| `req.inf.stg.03` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. |
+| `req.inf.stg.04` | Storage | The Architecture **should** make the immutable images available via location independent means. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable VIM instance storage |
+| `req.inf.stg.06` | Storage | The Architecture **should** make shared storage available to tenants as network-based File or Object Storage and support very large file needs. |
+| `req.inf.stg.07` | Storage | The Architecture **should** allow to use externally provided large archival storage for its Backup / Restore / Archives needs. |
+| `req.inf.stg.08` | Storage | The Architecture **should** make available all non-host OS / Hypervisor / Host systems storage as network-based Block, File or Object Storage for tenant/management consumption. |
+| `req.inf.ntw.01` | Network | The Architecture **must** provide virtual network interfaces to VM instances |
+| `req.inf.ntw.02` | Network | The Architecture **must** include an SDN integration to support provisioning of network services from host-based OpenStack Neutron networking VTEPs to the Border Edge based VRFs. |
+| `req.inf.ntw.03` | Network | The Architecture **must** Localize intra-host traffic to provide low latency, high throughput, and resiliency. |
+| `req.inf.ntw.04` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.05` | Network | The Architecture **must** allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay - VXLAN or Geneve). |
+| `req.inf.ntw.06` | Network | The Architecture **should** support Distributed Virtual Routing (DVR) to allow compute nodes to route traffic efficiently |
+| `req.inf.ntw.07` | Network | The Architecture **must** provide low latency and high throughput (do we need to quantify it?). |
+| `req.inf.ntw.08` | Network | The NFVI Network Fabric **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems |
+| `req.inf.ntw.09` | Network | The NFVI Network Fabric **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.10` | Network | The NFVI Network Fabric **must** be capable of supporting carrier-specific VNF workloads. |
+| `req.inf.ntw.11` | Network | The NFVI Network Fabric **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.12` | Network | The NFVI Network Fabric and Network Operating System **must** provide network operational visibility through alarming and streaming telemetry services for operational management, engineering planning, troubleshooting, and network performance optimisation. |
+| `req.inf.ntw.13` | Network | A SDN solution **must** be provided . |
```

Comment:
Duplicate / overlapping with req.inf.ntw.02. 

General question: is Neutron API with OpenStack out of the box implementation (OVS based) a suitable SDN solution? If yes, than "must" is ok, if not than "should" is more appropriate.

Reply:
In my view OVS can qualify for SDN so with this assumption we can change to "must", good point.

Reply:
@hdamker, Neutron APIs + OVS is considered a SDN solution. Therefore, I think "must" is OK. (do you think we need to clarify this?)

Reply:
> @hdamker, Neutron APIs + OVS is considered a SDN solution. Therefore, I think "must" is OK. (do you think we need to clarify this?)

@rabi-abdel, I have a different view ( not soo soo much:)) , SDN add more feature and benefit for the infrastructure which is missed from Vanilla OVS+ Neutron especially for SRIOV use case that needs Hardware touching, such as Automation, enhanced security enforcement, service channing,...etc  , so i will go with " Should"

Reply:
Understood. during the meeting, the consensus was it will be better to remove it completely as it is not adding too much value.